src/EditorFeatures/VisualBasic/UseAutoProperty/UseAutoPropertyAnalyzer.vb

```diff
@@ -0,0 +1,138 @@
+﻿Imports System.Collections.Concurrent
+Imports System.ComponentModel.Composition
+Imports System.Threading
+Imports Microsoft.CodeAnalysis.Diagnostics
+Imports Microsoft.CodeAnalysis.UseAutoProperty
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UseAutoProperty
+    <Export>
+    <DiagnosticAnalyzer(LanguageNames.VisualBasic)>
+    Friend Class UseAutoPropertyAnalyzer
+        Inherits AbstractUseAutoPropertyAnalyzer(Of PropertyBlockSyntax, FieldDeclarationSyntax, ModifiedIdentifierSyntax, ExpressionSyntax)
+
+        Private ReadOnly semanticFacts As New VisualBasicSemanticFactsService()
+
+        Protected Overrides Function SupportsReadOnlyProperties(compilation As Compilation) As Boolean
+            Return True
```

Comment:
Is this supposed to be ReadOnlyAutoProperties? If so, that's only supported in VB version 14.